### DISCUSSION TITLE
[Feedback] [UI] Copy button for code block blocks code display

### DISCUSSION THREAD

**Discussion by MasterInQuestion** [community/community#106841] [Other Feature Feedback, Questions, & Ideas]
https://github.com/orgs/community/discussions/29031
    https://github.com/orgs/community/discussions/7733

    The Copy button may cover the code texts: causing unwanted obstruction.

    My fix to the issue:
(uBlock Origin)
[[
```adblock
	github.com##:is( div, article ) > pre:style( tab-size: 4 !important; )
	github.com##div > div.zeroclipboard-container:style( opacity: 0.08 !important; )
	github.com##div > div.zeroclipboard-container:hover:style( opacity: 1 !important; )
```
]]
    Making the Copy button default mostly transparent, and opaque only when [":hover"](https://developer.mozilla.org/en-US/docs/Web/CSS/:hover).
    Additionally also overridden the at times annoying default "tab-size: 8" [1].

    See also: https://github.com/orgs/community/discussions/48072#discussioncomment-8390798

[ [1]
    [https://github.com/orgs/community/discussions?discussions_q=in:title+tab-size](https://github.com/orgs/community/discussions?discussions_q=in:title+tab-size)

    Adding "white-space: pre-wrap; word-wrap: break-word" [2] as in:
    [ https://github.com/madler/zlib/issues/910](https://github.com/madler/zlib/issues/910)
    ; may also address the line-wrap issue. ]
[ [2]
    Note ["!important"](https://developer.mozilla.org/en-US/docs/Web/CSS/important) has to be stated for each statement:
    uBO's style injection didn't give desired priority ("specificity"), as of 1.58.0. ]

**Comment by gil-son** [comment]
Where you wish to stylize? Markdown? HTML?

Please give me more details to support you

**Reply by MasterInQuestion** [reply]
Thank you.
    But I locally addressed the issue already.

    It's design issue of the UI on ["github.com"](https://github.com/). (HTML, if you must categorize)
